### PR TITLE
Delete cards lists

### PR DESCRIPTION
**Pull request by leenaamrutha1**

### CODE REVIEW

File: features/create_list_cards.feature [hunk 1]
```diff
@@ -14,48 +14,83 @@ Feature: create board in trello application
...
+    Then Validate menu header is displayed
+    Then Validate close board option is displayed
+    When Click on close board option
+    Then Validate close board pop up is displayed
+    Then Validate close board header "Close board?" is displayed
+    Then Validate close board content "find and reopen closed" is displayed
+    Then Validate close button is displayed
+    When Click on the close button
+    Then Validate randomly board name is closed text is displayed
+    Then Validate re open board button "Reopen board" is displayed
```
Comment (ManojSavaram) [inline features/create_list_cards.feature]: Then Validate "Reopen board" button is displayed in the Delete boards page.

File: features/create_list_cards.feature [hunk 2]
```diff
@@ -14,48 +14,83 @@ Feature: create board in trello application
...
+    Then Validate permanently delete board button "Permanently delete board" is displayed
```
Comment (ManojSavaram) [inline features/create_list_cards.feature]: Follow same approach as 48 line.

File: features/create_list_cards.feature [hunk 3]
```diff
@@ -14,48 +14,83 @@ Feature: create board in trello application
...
+    When Click on permanently delete board button
+    Then Validate delete board pop up is displayed
+    Then Validate delete board header "Delete board?" is displayed
+    Then Validate delete board content "cards and actions will be deleted" is displayed
+    Then Validate delete button is displayed
+    When Click on delete button
+    Then Validate board deleted pop up is displayed
```
Comment (ManojSavaram) [inline features/create_list_cards.feature]: Add this step:
Then Validate board delete pop-up content :"" is displayed

File: features/create_list_cards.feature [hunk 4]
```diff
@@ -14,48 +14,83 @@ Feature: create board in trello application
...
+    Then Validate delete random board name in your work space section
 
+  Scenario: deleting the cards and lists in the board
+    When Click on create button "Create"
```
Comment (ManojSavaram) [inline features/create_list_cards.feature]: Follow the 48 step approach.

File: features/create_list_cards.feature [hunk 5]
```diff
@@ -14,48 +14,83 @@ Feature: create board in trello application
...
+    Then Validate create form
+    When Click on create board button "Create board"
```
Comment (ManojSavaram) [inline features/create_list_cards.feature]: Follow the 48-step approach.

File: pages/add_a_list_card_page.py [hunk 6]
```diff
@@ -24,6 +24,7 @@ def enter_list_title(self,input):
         list_title.send_keys(input)
 
     def click_on_add_a_card_button(self, input):
+        time.sleep(3)
```
Comment (ManojSavaram) [inline pages/add_a_list_card_page.py]: Use dynamic waits

File: features/create_list_cards.feature (hunk 1 above)
Comment (leenaamrutha1) [reply features/create_list_cards.feature]: Hi @ManojSavaram 
I have worked on the review comments in this PR "https://github.com/OW-Practice/selenium-cucumber/pull/4"
image

File: features/create_list_cards.feature (hunk 2 above)
Comment (leenaamrutha1) [reply features/create_list_cards.feature]: Hi @ManojSavaram
I have worked on the review comments in this PR "https://github.com/OW-Practice/selenium-cucumber/pull/4"
image

File: features/create_list_cards.feature (hunk 3 above)
Comment (leenaamrutha1) [reply features/create_list_cards.feature]: Hi @ManojSavaram
I have worked on the review comments in this PR "https://github.com/OW-Practice/selenium-cucumber/pull/4"
image

File: features/create_list_cards.feature (hunk 4 above)
Comment (leenaamrutha1) [reply features/create_list_cards.feature]: Hi @ManojSavaram
I have worked on the review comments in this PR "https://github.com/OW-Practice/selenium-cucumber/pull/4"
image

File: features/create_list_cards.feature (hunk 5 above)
Comment (leenaamrutha1) [reply features/create_list_cards.feature]: Hi @ManojSavaram
I have worked on the review comments in this PR "https://github.com/OW-Practice/selenium-cucumber/pull/4"
image

**Review by ManojSavaram — APPROVED** [review]

**Review by ManojSavaram — APPROVED** [review]

**Comment by ManojSavaram** [comment]
This PR is modified as per the comments. Please merge it.

**Review by ManojSavaram — APPROVED** [review]

**Comment by ManojSavaram** [comment]
Updated as per review comments. Please merge it

**Pull request merged**